src/commonMain/kotlin/komem/litmus/LitmusOutcome.kt

```diff
@@ -11,9 +11,9 @@ data class LitmusOutcomeInfo(
 )
 
 data class LitmusOutcomeSetupScope(
-    var accepted: Set<LitmusOutcome> = emptySet(),
-    var interesting: Set<LitmusOutcome> = emptySet(),
-    var forbidden: Set<LitmusOutcome> = emptySet(),
+    var accepted: Set<LitmusOutcome> = mutableSetOf(),
+    var interesting: Set<LitmusOutcome> = mutableSetOf(),
+    var forbidden: Set<LitmusOutcome> = mutableSetOf(),
```

Comment:
I don't like the verbosity of writing 
```kotlin
accepted = setOf(
  listOf(1, 0), listOf(0, 1), listOf(0, 0)
)
```
So this is just an attempt of allowing more freedom. In other words, WIP

Reply:
So what would be the alternative with `MutableSet`?

Like this?
```
    accepted
        .add(listOf(1, 0))
        .add(listOf(0, 1))
        .add(listOf(0, 0))
```

It would actually make sense, if we split `LitmusOutcomeSetupScope` into two classes (similarly as `LitmusTest` and `LitmusTestScope`) --- one mutable class to be used as a builder, and one immutable class to store the result of the builder.